pallets/dapp-staking-v3/src/test/testing_utils.rs

```diff
@@ -86,9 +86,9 @@ impl MemorySnapshot {
     /// Returns locked balance in dApp staking for the specified account.
     /// In case no balance is locked, returns zero.
     pub fn locked_balance(&self, account: &AccountId) -> Balance {
-        self.ledger
```

Comment:
can we return `total_locked_amount`?

Reply:
We could, but seeing how I used the same method in testing_utils & in pallet code, I decided to intentionally _not_ re-use the same method again.